bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/GCData.java

```diff
@@ -47,6 +47,7 @@ public final class GCData {
 	public float[] lineDashes;
 	public float lineMiterLimit = 10;
 	public int alpha = 0xFF;
+	public int nativeDeviceZoom;
```

Comment:
Is `deviceZoom` the correct word here? The value is not tied to the device but to the consumer (widget or rather it's shell), isn't it? This applies to all field, parameter, method ... names in this PR.

Reply:
Let's name it nativeZoom. It contains the nativeZoom of the monitor on which GC is going to paint.